### DISCUSSION TITLE
Qwen3_30B_A3B获取logit时异常

### DISCUSSION THREAD

**Discussion by maxkura** [QwenLM/Qwen3#1710] [Q&A]
如图为获得的logit，所有值均相同且异常
<img width="764" height="246" alt="download" src="https://github.com/user-attachments/assets/0ced62d1-6dd4-4ba2-9043-0ca5d77d3755" />

加载精度为bf16
输入token_id正常
获取logit的代码为：lm_logits = model(inputs).logits
请问为什么获取logit时会产生这样的异常？或者说，获取qwen3的logit方式是什么

**Comment by jklj077** [comment]
一般是没显示全，显全了里面就有不同的了。
